server/pages/index.js

```diff
@@ -307,8 +307,8 @@ module.exports = function() {
 
 	// redirects to handle old newdash formats
 	app.use( '/sites/:site/:section', function( req, res, next ) {
+		const sections = [ 'posts', 'pages', 'sharing', 'upgrade', 'checkout', 'change-theme' ];
```

Comment:
This (rightly) gives me a `no-shadow` warning from ESLint, so let's use a different var name here. What about `redirectedSections`?